src/lib/utils.js

```diff
@@ -7,12 +7,26 @@ import { browser } from "$app/env";
 import { token } from "$lib/auth";
 import { defaultAcceptHeader } from "$lib/utils/api";
 
-export function markdownToHTML(md) {
+export function markdownToHTML(md, titleLevel) {
   const converter = new showdown.Converter({
     extensions: [
       () => ({
         type: "output",
         filter(html) {
+          if (titleLevel && titleLevel === 3) {
```

Comment:
- il faut commencer à h1, pas à h2; les boutons de l'éditeur commencent à h2, mais les gens copie-colle de l'HTML dedans, et on ne controle pas le contenu
- trop de copier coller :sweat_smile: 
un truc genre 
```
for (let lvl = 1; lvl < 7; ++level) {
  const pattern = new RegExp(`h${lvl}`, gi);
  html = html.replace(pattern, `h${lvl+1}`;
}
```
me paraitrait mieux…
- je trouve la regexp un peu trop large... tous les hN du texte vont être modifiés, même si ce n'est pas un tag... A minima, je testerais "<hN" plutot. 
- mais surtout, en lisant ça, je me disais que le mieux c'est de voir si on ne peut pas le faire proprement au niveau de showdown, et :tada: c'est prévu regarde `headerLevelStart` sur [headerLevelStart](https://showdownjs.com/docs/available-options/)
Par contre ça n'a sans doute plus de sens de faire des titres 2 et 3 dans l'editeur, autant faire des 1 et 2 directement...

Quand à l'ajout du nofollow qui est fait juste en dessous, on gagnerait à le faire avec insane je pense: https://github.com/bevacqua/insane#filter

Reply:
En effet, bien vu l'option `headerLevelStart` (je n'avais pas consulté la doc...) - et du coup, j'ai remis le niveau de base à `h1` dans l'éditeur pour ne pas perdre un niveau à chaque fois...

Reply:
Par contre, pour l'ajout de `nofollow`, `insane` ne permet pas de le faire (contrairement à `sanitize-html` dont il s'inspire qui dispose de `transformTags`)

Reply:
Pour le nofollow, c'est la phrase suivante qui m'a fait pensé que ça devrait être possible: "Note that you are allowed to change the attrs, and even add new ones, transforming the output."
Dans tous les cas, ça sera pour une autre PR!

Reply:
Mais du coup... est-ce qu'on les passe aussi en target blank/noopener? :thinking: 
Pas d'avis là dessus, c'est toi qui vois ;-)